tally_ho/apps/tally/views/data/race_list_view.py

```diff
@@ -13,6 +13,9 @@ def get(self, *args, **kwargs):
 
         ballots = paging(ballots, self.request)
 
+        for record in ballots:
+            record['document_name'] = record['document'].rsplit('/', 1)[-1]
```

Comment:
It's getting the file name since document is being returned as a path variable. Will use the `[pathlib]` and create a `Path` instance then use `.name` to get the file. Will do this in the `Ballot` class.